turn-on.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const btnTurn = document.querySelector('.button');
+const theme = document.querySelector('.wrapper'); 
+const lastChanchText = document.querySelector('.last-turn');
+let lastChanchTime = localStorage.getItem('lastChanch');
```

Comment:
```suggestion
let lastChangeTime = localStorage.getItem('lastChange');
```
Тут та далі - поправте помилки в назвах